app/controllers/articles_controller.rb

```diff
@@ -1,8 +1,8 @@
 class ArticlesController < ApplicationController
-  
   include Pagination
 
   def index
+    @profile = current_profile
```

Comment:
The current profile will already be assigned to `@profile` in the `current_profile` method. Since `ArticlesController` inherits from `ApplicationController` it will be in scope already